.github/workflows/build.yml

```diff
@@ -311,12 +311,27 @@ jobs:
       run: |
         mkdir -p allure-results/history
         cp -r build-history-data/cht-core/allure/${{ env.JOB_NAME }}/allure-report/history allure-results | true
+    - name: Cache Chrome 90 Installer
```

Comment:
We should only be trying to download Chrome 90 on CI jobs that require this browser. There is a line below, which you have deleted, that achieves this: 

> if: ${{matrix.chrome-version=='90'}}

This code also looks like is failing all builds now because it fails to download the Chrome 90 for some reason.